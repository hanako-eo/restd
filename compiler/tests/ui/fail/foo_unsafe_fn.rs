#[compiler::item("foo_fn_unsafe")]
fn foo() {}

fn main() {}
