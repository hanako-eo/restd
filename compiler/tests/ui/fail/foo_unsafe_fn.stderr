error: `foo_fn_unsafe` item must be an unsafe function
 --> tests/ui/fail/foo_unsafe_fn.rs:2:1
  |
2 | fn foo() {}
  | ^^^^^^^^^^^
